dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryMultiAppTest.java

```diff
@@ -95,8 +103,6 @@ public static void setup() throws Exception {
 
         ShrinkHelper.exportAppToServer(server, multiapp2, SERVER_ONLY);
 
-        server.addEnvVar("OTEL_SDK_DISABLED", "false");
-        server.addEnvVar("OTEL_BSP_SCHEDULE_DELAY", "100");
         server.startServer();
```

Comment:
These changes presumably relate to the issues you were having with pre-2.0 versions of mpConfig.